src/agora/docoding.cpp

```diff
@@ -152,6 +162,10 @@ EventData DoDecode::Launch(size_t tag) {
   bblib_ldpc_decoder_5gnr(&ldpc_decoder_5gnr_request,
                           &ldpc_decoder_5gnr_response);
 
+  // Descramble the decoded buffer
+  WlanScramble((int8_t*)decoded_buffer_ptr, cfg_->NumBytesPerCb(),
```

Comment:
```suggestion
  WlanScramble(reinterpret_cast<int8_t*>(decoded_buffer_ptr), cfg_->NumBytesPerCb(),
```

Reply:
Should we have a "descramble function"?  Ie a new function called WlanDescramble.  The internal implementation of that function would just be a call to Scramble with the inputs reversed (i am just guessing here)